Documentation/running-lkvm-stage1.md

```diff
@@ -0,0 +1,97 @@
+[    0.253977] etcd[71]: 2015/08/04 10:47:04 etcd: listening for peers on http://localhost:7001
+[    0.253977] etcd[71]: 2015/08/04 10:47:04 etcd: listening for client requests on http://localhost:2379
+[    0.254977] etcd[71]: 2015/08/04 10:47:04 etcd: listening for client requests on http://localhost:4001
+[    0.254977] etcd[71]: 2015/08/04 10:47:04 etcdserver: datadir is valid for the 2.0.1 format
+[    0.255977] etcd[71]: 2015/08/04 10:47:04 etcdserver: name = default
+[    0.255977] etcd[71]: 2015/08/04 10:47:04 etcdserver: data dir = default.etcd
+[    0.255977] etcd[71]: 2015/08/04 10:47:04 etcdserver: member dir = default.etcd/member
+[    0.255977] etcd[71]: 2015/08/04 10:47:04 etcdserver: heartbeat = 100ms
+[    0.255977] etcd[71]: 2015/08/04 10:47:04 etcdserver: election = 1000ms
+[    0.256977] etcd[71]: 2015/08/04 10:47:04 etcdserver: snapshot count = 10000
+[    0.256977] etcd[71]: 2015/08/04 10:47:04 etcdserver: advertise client URLs = http://localhost:2379,http://localhost:4001
+[    0.256977] etcd[71]: 2015/08/04 10:47:04 etcdserver: initial advertise peer URLs = http://localhost:2380,http://localhost:7001
+[    0.256977] etcd[71]: 2015/08/04 10:47:04 etcdserver: initial cluster = default=http://localhost:2380,default=http://localhost:7001
+[    0.268975] etcd[71]: 2015/08/04 10:47:04 etcdserver: start member ce2a822cea30bfca in cluster 7e27652122e8b2ae
+[    0.270975] etcd[71]: 2015/08/04 10:47:04 raft: ce2a822cea30bfca became follower at term 0
+[    0.270975] etcd[71]: 2015/08/04 10:47:04 raft: newRaft ce2a822cea30bfca [peers: [], term: 0, commit: 0, applied: 0, lastindex: 0, lastterm: 0]
+[    0.270975] etcd[71]: 2015/08/04 10:47:04 raft: ce2a822cea30bfca became follower at term 1
+[    0.278974] etcd[71]: 2015/08/04 10:47:04 etcdserver: added local member ce2a822cea30bfca [http://localhost:2380 http://localhost:7001] to cluster 7e27652122e8b2ae
+[    1.573068] etcd[71]: 2015/08/04 10:47:05 raft: ce2a822cea30bfca is starting a new election at term 1
+[    1.574244] etcd[71]: 2015/08/04 10:47:05 raft: ce2a822cea30bfca became candidate at term 2
+[    1.575796] etcd[71]: 2015/08/04 10:47:05 raft: ce2a822cea30bfca received vote from ce2a822cea30bfca at term 2
+[    1.576794] etcd[71]: 2015/08/04 10:47:05 raft: ce2a822cea30bfca became leader at term 2
+[    1.577860] etcd[71]: 2015/08/04 10:47:05 raft.node: ce2a822cea30bfca elected leader ce2a822cea30bfca at term 2
+[    1.587885] etcd[71]: 2015/08/04 10:47:05 etcdserver: published {Name:default ClientURLs:[http://localhost:2379 http://localhost:4001]} to cluster 7e27652122e8b2ae
+```
+
+## What doesn't work
+
+LKVM support is in an very early stage so basic features don't work yet:
+
+* Volumes support [#1176](https://github.com/coreos/rkt/pull/1176)
+* Network support [#1219](https://github.com/coreos/rkt/pull/1219)
+
+## How does it work?
+
+It leverages the work done by Intel with their [Clear Containers system](https://lwn.net/Articles/644675/).
```

Comment:
OK, I'll write something along

"Stage1 contains a Linux kernel that is executed under LKVM.
This kernel will then start systemd, which in turn will start the applications in the pod."